frontend/src/models/groupsModel.ts

```diff
@@ -57,7 +57,7 @@ export const groupsModel = kea<groupsModelType>({
                         plural: groupType.name_plural || `${groupType.group_type}(s)`,
                     }
                 }
-                return { singular: 'person', plural: 'people' }
+                return { singular: 'user', plural: 'users' }
```

Comment:
Could this have some unintended side-effects?

Reply:
I don't believe so (think it's just a matter of making the naming more consistent), but gonna tag @macobo to be sure

Reply:
The nomenclature here has gone back and forth 100 times at this point, with @paolodamico, @clarkus and @EDsCODE all being involved. I _believe_ the correct answer is that "persons" is correct and should be the nomenclature through the app.

Reply:
So apparently "user" vs. "person" is a contentious topic. Reverted to "person" being the default here to avoid affecting other places. Instead I added an optional arg `deferToUserWording` so that smart names can still be consistent with query definition selectors.